### PR TITLE
Update taxonomy.simple.yaml

### PR DESCRIPTION
**Pull request by morchickit**
- Remove duplicate of stage of care
- Adding transportation 
- Adding Help to pay for transport under money 
- Adding Ex prisoners under crime 
- Adding תושבים חוזרים
- Adding consumer rights protection under money
- Adding Crime Prevention under care

### CODE REVIEW

File: taxonomy.simple.yaml [hunk 1]
```diff
@@ -235,6 +240,7 @@
       - Business Loans
       - Personal Loans
     - Tax Preparation
+    - Cunsomer rights protection
```
Comment (akariv) [inline taxonomy.simple.yaml:243]: Consumer Rights Protection (typo)

File: taxonomy.simple.yaml [hunk 2]
```diff
@@ -315,13 +321,15 @@
     - Refugees
     - Residents
     - Statusless
+    - Returned citizens
```
Comment (akariv) [inline taxonomy.simple.yaml:324]: Returning Residents

File: taxonomy.simple.yaml [hunk 3]
```diff
@@ -485,6 +492,7 @@
     - Natural Disaster Survivors
     - Prostition Survivors
     - Trauma Survivors
+    - Offence survivors
```
Comment (akariv) [inline taxonomy.simple.yaml:495]: Offense survivors?

**Comment by morchickit** [comment]
תיקנתי פה - f43bd72e43bf9d48ec24288db50ae5cb27ab4f8f

**Pull request merged**